src/components/Contacts/ContactDetails/ContactDetailsHeader/ContactHeaderSection/ContactHeaderStatusSection.tsx

```diff
@@ -22,9 +22,7 @@ const TextSkeleton = styled(Skeleton)(({}) => ({
 export const ContactHeaderStatusSection = ({
   loading,
   contact,
-}: Props): ReactElement => {
-  const { t } = useTranslation();
-
+}: Props): ReactElement | null => {
```

Comment:
Could you do ```ContactHeaderStatusSection: React.FC<Props>``` instead? I believe that type returns ```ReactElement | null```. Probably pros and cons to either 🤷‍♂️